feat(start_nominating): add section how to change nominees (#35)
galacticcouncil/HydraDX-docs: docs/start_nominating.md

<div style={{textAlign: 'center'}}>
  <img src={useBaseUrl('/nominator-guide/nominate-validator-4.png')} />

> </div>  

Reviewer: ```suggestion
</div>
```

Author: Double-space is a linebreak in markdown (<br>)

:::note

Make sure that you revisit your nominations once in a while. It is possible that some of your validators change their commission percentage which will have a negative impact on your rewards. By checking the status of your nominations frequently you will be able to react by updating the list of your nominated validators.

:::

> ## 04 Change Nominees

Reviewer: Adjust your nominations

(let's not use Camel Case to remain consistent with other titles)

> There may be several circumstances where you'd like to change your nominations or remove specific ones.  

Reviewer: If some of your validators become oversubscribed or change their commission, you might want to adjust your nominations.

To do so, open Polkadot/apps and navigate to:

This could be due to oversubscription of a validator or if he changes the commission rate.  

To do so open Polkadot/apps and navigate over to:  
*Network* > *Staking* > *Account actions*

> Hit the three dots at the end of your account details and click _Set nominees_.  

Reviewer: ```suggestion
Click on the three dots next to your account details ans select _Set nominees_.
```

<div style={{textAlign: 'center'}}>
  <img src={useBaseUrl('/nominator-guide/nominate-set-nominees.png')} />
</div>

> In the following window, that may already feel familiar, you can remove validators and/or add new ones.  

Reviewer: ```suggestion
You will see the nominating UI where you can adjust your nominations by removing and adding validators.
```

> This step can be done on the fly, there is no need to stop nominating. It will be respected for the next era.  

Reviewer: ```suggestion
Adjusting your nominations can be done on the fly, there is no need to stop nominating. The changes will be applied when the following era begins (24h).
```
